### ISSUE TITLE
3.2-calc-Alia Borroho

### ISSUE THREAD

**Issue by aliaborroho** [TIY-LR-FEE-2015-June/assignments#89]
To Do:
- [x] Fallow along with class.
- [x] Deploy to GH pages.

**Comment by aliaborroho** [comment]
Pull Request:

https://github.com/aliaborroho/3.2-calc/pull/1

GH Pages:

http://aliaborroho.github.io/3.2-calc/

**Comment by aliaborroho** [comment]
@ryan I believe this was to follow along with the class, but it is labeled needs work???

**Comment by rtablada** [comment]
@aliaborroho yes the goal was to follow along with class on this assignment. But understanding the error: `Cannot set property 'onclick' of null` is part of the assignment and understanding of JS.

Let's fix this Monday morning to check and update the issue.

**Comment by aliaborroho** [comment]
Yes, thanks Ryan😉
